File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) 
    {int i=0,j=0;
    int len=nums.length;
    for( i=1;i<len;i++)
      {for( j=i;j<len;j++)
          {if(nums[j-i]+nums[j]==target)
            {
                  return new int[] {j-i,j};
            }
            }
      } 
        return new int[]{}; 
    }
}

File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans=[]
        for i in range(1,len(nums)):
            for j in range(i,len(nums)):
                sum=nums[j-i]+nums[j]
                if sum==target :
                    return [j-i,j]
        return ans
        

        